smart-freight-center/sfc-backend
# 14 create GitHub actions workflow to build the backend docker image

## Description
This resolves #14 

### How to test it
- Testing the docker file:
  - Run `docker build . -t sfc-backend` in the root folder.
  - Run `docker run -e NODE_ENV=production -p 3000:3000 sfc-backend` then open `localhost:3000/health` in the browser to verify that the health check is running.
- Testing the github action workflow
  - Verify that the github action workflow was run successfully.
  - Go to the ECR repository on the SFC AWS account and verify that the a new image was pushed.
